packages/amplify_analytics_pinpoint/pubspec.yaml

```diff
@@ -4,8 +4,8 @@ version: 0.2.1
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_analytics_pinpoint
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
-  flutter: ^1.20.0
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.20.0"
```

Comment:
Nit: Can we drop the quotes on non-ranges to be consistent with Flutter/Dart docs?

Reply:
discussed offline, but we **think** this needs quotes based on how my editor was coloring it